manifest/autoversion/stateutils_test.go

```diff
@@ -0,0 +1,80 @@
+package autoversion
```

Comment:
Basically copied from:
https://github.com/cashapp/hermit/blob/c375b97e9b2515d79f0ba92dec2c78c91781b1a6/state/stateutils_test.go

Reply:
If this is being shared it should be factored into a `statetest` package under `state`

Reply:
Once that happens, are there still circular imports?